functional.py

```diff
@@ -0,0 +1,37 @@
+
+def sequential_map(*args):
+  list_of_values = args[-1]
+  def funccc(x):
+    for arg in args[:-1]:
+      x = list(map(arg, x))
+    return x
+  return funccc(list_of_values)
```

Comment:
1. Создание внутренней функции не оправдано, так как она вызывается сразу после создания
2. Превратить элемент в список можно только в самом конце, map можно передавать в другие map'ы
3. Можно удобно распаковать args сразу в две переменные
4. Отступы должны быть кратны 4 пробелам (это относится ко всей работе)
```suggestion
    *functions, list_of_values = args
    for func in functions:
        list_of_values = map(func, list_of_values)
    return list(list_of_values)
```